Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchableUnboundedPlane.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Add a NearInteractionTouchableUnboundedPlane to your scene and configure a touchable volume
+    /// in order to get PointerDown and PointerUp events whenever a PokePointer touches the
+    /// attached collider through this surface.
+    /// </summary>
+    public class NearInteractionTouchableUnboundedPlane : BaseNearInteractionTouchable
+    {
+#if UNITY_EDITOR
+        private readonly Color handleColor = Color.white;
```

Comment:
You could actually move this to NearInteractionTouchableUnboundedPlaneEditor.cs and move into an Editor folder to remove the #if UNITY_EDITOR. Can do in a later PR.